app/src/main/java/org/simple/clinic/sms/SmsReader.kt

```diff
@@ -0,0 +1,15 @@
+package org.simple.clinic.sms
+
+import io.reactivex.Single
+
+sealed class SmsReadResult {
+
+  data class SmsReadSuccess(val string: String) : SmsReadResult()
```

Comment:
Do you want to remove the prefix from these two classes? Because `SmsReadResult.SmsReadSuccess` sounds redundant?